CONTRIBUTING.md

```diff
@@ -14,6 +14,62 @@ Thanks for taking the plunge!
   (`Pkg.add(name="DataFrames", rev="main")`) is a good gut check and can streamline the process,
   along with including the first two lines of output from `versioninfo()`
 
+## Modifying an existing docstring in `src/`
+
+All docstrings are written inline above the methods or types they are associated with and can
+be found by clicking on the `source` link that appears below each docstring in the HTML file.
+The steps needed to make a change to an existing docstring are listed below:
+
+* Find the docstring in `src/`;
+* Update the text in the docstring;
+* run `make docs` from the root directory;
```

Comment:
As discussed - why do you write `make dos`? I would assume `julia --project=docs docs/make.jl` run from the top-level folder of the package? But maybe I am missing something here?